Header header                       # header.frame_id is not
                                    #   used for this message

uint16 system_id
